master/internal/db/postgres_test_utils.go

```diff
@@ -198,8 +198,9 @@ func RequireMockUser(t *testing.T, db *PgDB) model.User {
 		PasswordHash: null.NewString("", false),
 		Active:       true,
 	}
-	_, err := db.AddUser(&user, nil)
-	require.NoError(t, err, "failed to add user")
+	// TODO CAROLINA: remove import cycle
+	// _, err := user.AddUser(&user, nil)
+	// require.NoError(t, err, "failed to add user")
```

Comment:
blocker: I don't know how to remove the import cycle here -- db can't import user, because user imports db

Reply:
it would cause another circular import since all the intg tests in the `db` package reference `RequireMockUser` at some point or another

Reply:
huh, yeah, i was worried about that. this is an annoying one to fix.